examples/bigcommerce/package.json

```diff
@@ -19,6 +19,7 @@
     "@types/react": "18.0.15",
     "@types/react-dom": "18.0.6",
     "autoprefixer": "^10.4.8",
+    "eslint": "^8.22.0",
```

Comment:
eslint is included in next.js but using the config packages below warns that eslint isn't installed.

Just added eslint to remove the warning

Reply:
I'm surprised that's the case. Does this mean that a fresh `create-next-app` installation will have those warnings?